dashboard/app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -222,6 +248,15 @@ def prepare_locale_cookie(user)
     end
   end
 
+  def email_already_taken_redirect(provider:, found_provider:, email:)
+    # Note that @user.email is populated by User.from_omniauth even for students
```

Comment:
this comment can be removed or changed now that this functionality has been pulled into a helper